src/routes/MemberProfile/MemberProfile.js

```diff
@@ -33,60 +38,106 @@ export default class MemberProfile extends React.Component {
 
   selectTab = currentTab => this.setState({ currentTab })
 
-  blockUser = (personId) => () => {
+  blockUser = personId => {
     if (window.confirm(BLOCK_CONFIRM_MESSAGE)) {
       this.props.blockUser(personId).then(this.props.goToPreviousLocation)
     }
   }
 
   render () {
     if (this.props.error) return <Error>this.props.error</Error>
-    if (!this.props.person) return <Loading />
+    if (this.props.personLoading) return <Loading />
+
     const {
-      loading,
+      contentLoading,
       person,
       currentUser,
-      routeParams
+      routeParams,
+      push
     } = this.props
-    const {
-      bannerUrl,
-      bio,
-      tagline
-    } = person
     const { currentTab } = this.state
     const personId = routeParams.personId
-    const isMe = currentUser && currentUser.id === personId
+    const firstName = getFirstName(person)
+    const locationWithoutUsa = person.location && person.location.replace(', United States', '')
+    const isCurrentUser = currentUser && currentUser.id === personId
     const isAxolotl = AXOLOTL_ID === personId
-    const itemsMenuItems = [
-      { icon: 'Ex', label: 'Block this Member', onClick: this.blockUser(personId), hide: isMe || isAxolotl }
+    const actionMenuItems = [
+      { icon: 'Edit', label: 'Edit Profile Settings', onClick: () => push(currentUserSettingsUrl()), hide: !isCurrentUser },
+      { icon: 'Ex', label: 'Block this Member', onClick: () => this.blockUser(personId), hide: isCurrentUser || isAxolotl }
     ]
+    const contentDropDownItems = [
+      { label: 'Overview', title: `${firstName}'s recent activity`, component: RecentActivity },
+      { label: 'Posts', title: `${firstName}'s posts`, component: MemberPosts },
+      { label: 'Comments', title: `${firstName}'s comments`, component: MemberComments },
+      { label: 'Upvotes', title: `${firstName}'s upvotes`, component: MemberVotes }
+    ].map(contentDropDownitem => ({
+      ...contentDropDownitem, onClick: () => this.selectTab(contentDropDownitem.label)
+    }))
+    const currentContent = contentDropDownItems.find(contentItem => contentItem.label === currentTab)
+    const CurrentContentComponent = currentContent.component
 
     return <div styleName='member-profile'>
-      <ProfileBanner bannerUrl={bannerUrl}>
-        <ProfileNamePlate {...person} rightSideContent={<MemberActionsMenu items={itemsMenuItems} />} />
-      </ProfileBanner>
+      <div styleName='header'>
+        <div styleName='header-banner' style={bgImageStyle(person.bannerUrl)}>
+          <RoundImage styleName='header-member-avatar' url={person.avatarUrl} xlarge />
+          <h1 styleName='header-member-name'>{person.name}</h1>
+          {person.location && <div styleName='header-member-location'>
+            <Icon name='Location' styleName='header-member-location-icon' />
+            {locationWithoutUsa}
+          </div>}
+          {/* TODO: Do we still want to show the "Community manager" role? */}
```

Comment:
oh dang, forgot to work this out with aaron. i think it's a good thing to show but we dont have a design for it yet

Reply:
Ok

Reply:
This isn't going to make this iteration of the feature. There seemed to be some confusion about it's meaning anyway, so maybe we can wait for it to come back into discussion and create a ticket for it then.